src/spyglass/common/common_interval.py

```diff
@@ -79,26 +79,47 @@ def fetch_interval(self):
         if not len(self) == 1:
             raise ValueError(f"Expected one row, got {len(self)}")
         return Interval(self.fetch1())
+        
+    def plot_intervals(self, start_time=0, return_fig=False):
+        """
+        Parameters
+        ----------    
+        start_time: int (seconds)
+            set the 0 time for your interval comparison plot (for example, the first timepoint of s1)
+
+        Returns
+        -------
+        None
+            this function displays a timeline plot of the intervals you want to compare
+        """
+        interval_lists_df = pd.DataFrame(self)
```

Comment:
I'm thinking it would be a good idea to put some reasonable limits on this process to avoid trying to call plotting on thousands of cases

@edeno Any thoughts on an appropriate upper bound?

```suggestion
        if 1 < len(self) < 50:
            raise ValueError
        interval_lists_df = pd.DataFrame(self)
```

Reply:
I probably wouldn't hard code an upper bound but just warn the user if they are trying to plot a large number of intervals so they can kill it.